Edits, distribution, and requirements (#7)
Jjagg/MonoGame: Documentation/requirements.md

> # Requirements

Reviewer: ```suggestion
```

Depending on the [platform](Platforms.md) that you are targeting, MonoGame has different sets of requirements.

## For desktop platforms

> MonoGame requires a .Net Core SDK (3.1 or up) installation.

Reviewer: ```suggestion
MonoGame requires a .NET Core SDK (3.1 or up) installation.
```

> You can either install it [independently](https://dotnet.microsoft.com/download/dotnet-core), or by selecting the .Net Core payload when installing Visual Studio 2019 (version 15.4 and up required).

Reviewer: ```suggestion
You can either install it [independently](https://dotnet.microsoft.com/download/dotnet-core), or by selecting the .NET Core payload when installing Visual Studio 2019 (version 15.4 and up required).
```

> When it comes to IDE, Visual Studio 2019, Visual Studio Code, and Visual Studio 2019 for Mac are supported (alternatively, you can work directly from the CLI with your code editor of choice).

Reviewer: Hmm, I'm not sure about this line. We don't really do anything to support VS Code, so it's not more or less supported than, say, JetBrains Rider. Even for VS and VS for Mac we only provide templates.

Or should we mention this so we don't have to troubleshoot issues with other IDE's?

Author: Really don't know what's best here. I don't if removing Code will expose us to trivial questions like "is Code supported?" or if Code/CLI users are proficient enough to assume that it is. But yeah, the question is the same for any code editor.

Author: Maybe what we should be noting is that VS2017 and prior are not supported instead?

Desktop development is possible from any operating system.

## For UWP platforms

> MonoGame requires the latest Windows 10 SDK.

Reviewer: I guess it does not really need to be the latest version, but this is definitely the easiest way to put it without going into Windows 10 SDK versioning and hardcoding a version here that would no doubt be outdated in the future because we forget to update it.

Author: Yeah, I guess it's safest to put it that way and may avoid us to edit the doc if any new SDK releases or if the requirements change.
The latest SDK is always shipped within VS 2019 and it allows to backtarget any Windows 10 version (and I believe that store submissions have to be using a quite recent SDK anyway).

You can install it by selecting the Universal App payload when installing Visual Studio 2019.

Building and publishing for UWP is only supported with Visual Studio 2019.

UWP development is not possible from macOS or Linux.

## For mobile platforms

MonoGame requires either Xamarin.iOS or Xamarin.Android depending on the target.

> You can install either of them by selecting the Mobile App development payload when installing Visual Studio 2019 or Visual Studio 2019 for Mac.

Reviewer: ```suggestion
In Visual Studio you can install both by selecting the 'Mobile development with .NET' workload.
In Visual Studio for Mac you can install the iOS and Android workload separately.
```

Only Visual Studio 2019 or Visual Studio 2019 for Mac are supported in those contexts.

> Mobile development is not possible from Linux.

Reviewer: @harry-cpp will have something to say about this :p 

In all seriousness though, we shouldn't go into mobile development from Linux in our docs. This is good as it is.
